coderedcms/project_template/website/models.py

```diff
@@ -80,3 +80,4 @@ class Meta:
         verbose_name = 'Web Page'
 
     template = 'coderedcms/pages/web_page.html'
+    
```

Comment:
Exclude/undo this change. File already has newline at the end, does not need another, makes the commit/PR look like it changes more files than it actually does.